tap/api/api.go

```diff
@@ -150,21 +150,21 @@ func (e *Emitting) Emit(item *OutputChannelItem) {
 }
 
 type Entry struct {
-	Id           string                 `json:"id"`
-	Protocol     Protocol               `json:"proto"`
-	Capture      Capture                `json:"capture"`
-	Source       *TCP                   `json:"src"`
-	Destination  *TCP                   `json:"dst"`
-	Namespace    string                 `json:"namespace"`
-	Outgoing     bool                   `json:"outgoing"`
-	Timestamp    int64                  `json:"timestamp"`
-	StartTime    time.Time              `json:"startTime"`
-	Request      map[string]interface{} `json:"request"`
-	Response     map[string]interface{} `json:"response"`
-	RequestSize  int                    `json:"requestSize"`
-	ResponseSize int                    `json:"responseSize"`
-	ElapsedTime  int64                  `json:"elapsedTime"`
-	Rules        ApplicableRules        `json:"rules,omitempty"`
+	Id                 string                 `json:"id"`
+	ProtocolUniqueName string                 `json:"protocol"`
```

Comment:
`ProtocolId` is shorter and clearer in my opinion. For your consideration.

Reply:
hmmm i thought about naming it like that, but it's not really an id so i went with unique name

Reply:
What is the difference from an ID?